runtime/proto_errors.go

```diff
@@ -8,8 +8,17 @@ import (
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/grpclog"
 	"google.golang.org/grpc/status"
+	"github.com/grpc-ecosystem/grpc-gateway/runtime/internal"
```

Comment:
Could you run `goimports` on this file please? I think this import should be above the grpc ones.